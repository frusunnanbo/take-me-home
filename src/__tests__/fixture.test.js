const request = require('supertest');

const app = require('../app');

describe('The take-me-home app', () => {
  it('Responds to /', () => {
    return request(app)
      .get('/trips?startTime=2019-11-09T11:31')
      .expect(200)
      .then((response) => {
        expect(response.body).toMatchSnapshot();
      });
  })
});
