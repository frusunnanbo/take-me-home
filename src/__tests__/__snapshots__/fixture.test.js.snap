// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Take-me-home app returns some trips 1`] = `
Array [
  Object {
    "duration": Object {
      "hours": 2,
      "minutes": 16,
    },
    "legs": Array [
      Object {
        "end": "2019-11-09T13:33:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Åkers Runö station (Österåker kn)",
          "time": "12:57",
        },
        "stops": Array [
          "Åkers Runö station (Österåker kn)",
          "Täljö station (Österåker kn)",
          "Rydbo station (Österåker kn)",
          "Hägernäs station (Täby kn)",
          "Viggbyholm station (Täby kn)",
          "Galoppfältet station (Täby kn)",
          "Täby centrum station",
          "Roslags Näsby station (Täby kn)",
          "Enebyberg station (Danderyd kn)",
          "Djursholms Ekeby station (Danderyd kn)",
          "Bråvallavägen station (Danderyd kn)",
          "Djursholms Ösby station (Danderyd kn)",
          "Mörby station (Danderyd kn)",
          "Universitetet station (Stockholm kn)",
          "Stockholm Östra station",
        ],
        "type": "JNY",
      },
      Object {
        "end": "2019-11-09T13:51:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Tekniska Högskolan T-bana (Stockholm kn)",
          "time": "13:45",
        },
        "stops": Array [
          "Tekniska Högskolan T-bana (Stockholm kn)",
          "Stadion T-bana (Stockholm kn)",
          "Östermalmstorg T-bana (Stockholm kn)",
          "T-Centralen T-bana (Stockholm kn)",
        ],
        "type": "JNY",
      },
      Object {
        "end": "2019-11-09T14:10:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Stockholm City station",
          "time": "14:01",
        },
        "stops": Array [
          "Stockholm City station",
          "Stockholm Södra station",
          "Årstaberg station (Stockholm kn)",
          "Älvsjö station (Stockholm kn)",
        ],
        "type": "JNY",
      },
      Object {
        "end": "2019-11-09T14:29:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Älvsjö station (Stockholm kn)",
          "time": "14:25",
        },
        "stops": Array [
          "Älvsjö station (Stockholm kn)",
          "Turingevägen (Stockholm kn)",
        ],
        "type": "JNY",
      },
    ],
    "start": Object {
      "date": "2019-11-09",
      "time": "12:57",
    },
  },
  Object {
    "duration": Object {
      "hours": 2,
      "minutes": 25,
    },
    "legs": Array [
      Object {
        "end": "2019-11-09T14:03:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Åkers Runö station (Österåker kn)",
          "time": "13:27",
        },
        "stops": Array [
          "Åkers Runö station (Österåker kn)",
          "Täljö station (Österåker kn)",
          "Rydbo station (Österåker kn)",
          "Hägernäs station (Täby kn)",
          "Viggbyholm station (Täby kn)",
          "Galoppfältet station (Täby kn)",
          "Täby centrum station",
          "Roslags Näsby station (Täby kn)",
          "Enebyberg station (Danderyd kn)",
          "Djursholms Ekeby station (Danderyd kn)",
          "Bråvallavägen station (Danderyd kn)",
          "Djursholms Ösby station (Danderyd kn)",
          "Mörby station (Danderyd kn)",
          "Universitetet station (Stockholm kn)",
          "Stockholm Östra station",
        ],
        "type": "JNY",
      },
      Object {
        "end": "2019-11-09T14:39:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Tekniska Högskolan T-bana (Stockholm kn)",
          "time": "14:15",
        },
        "stops": Array [
          "Tekniska Högskolan T-bana (Stockholm kn)",
          "Stadion T-bana (Stockholm kn)",
          "Östermalmstorg T-bana (Stockholm kn)",
          "T-Centralen T-bana (Stockholm kn)",
          "Gamla Stan T-bana (Stockholm kn)",
          "Slussen T-bana (Stockholm kn)",
          "Mariatorget T-bana (Stockholm kn)",
          "Zinkensdamm T-bana (Stockholm kn)",
          "Hornstull T-bana (Stockholm kn)",
          "Liljeholmen T-bana (Stockholm kn)",
          "Midsommarkransen T-bana (Stockholm kn)",
          "Telefonplan T-bana (Stockholm kn)",
          "Hägerstensåsen T-bana (Stockholm kn)",
          "Västertorp T-bana (Stockholm kn)",
          "Fruängen T-bana (Stockholm kn)",
        ],
        "type": "JNY",
      },
      Object {
        "end": "2019-11-09T15:08:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Fruängen T-bana (Stockholm kn)",
          "time": "14:51",
        },
        "stops": Array [
          "Fruängen T-bana (Stockholm kn)",
          "Fredrika Bremers gata (Stockholm kn)",
          "Isbergavägen (Stockholm kn)",
          "Långbrohallen (Stockholm kn)",
          "Johan Skytteskolan (Stockholm kn)",
          "Fru Marias väg (Stockholm kn)",
          "Långbrovägen (Stockholm kn)",
          "Vältstigen (Stockholm kn)",
          "Långbrodalsvägen (Johan Skyttes väg) (Stockholm kn",
          "Skyttes plan (Stockholm kn)",
          "Älvsjögården (Stockholm kn)",
          "Älvsjö station (Stockholm kn)",
          "Turingevägen (Stockholm kn)",
        ],
        "type": "JNY",
      },
    ],
    "start": Object {
      "date": "2019-11-09",
      "time": "13:27",
    },
  },
  Object {
    "duration": Object {
      "hours": 2,
      "minutes": 13,
    },
    "legs": Array [
      Object {
        "end": "2019-11-09T13:54:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Täljöviken (Österåker kn)",
          "time": "13:34",
        },
        "stops": Array [
          "Täljöviken (Österåker kn)",
          "Svinninge Gård (Österåker kn)",
          "Svinninge tomtområde (Österåker kn)",
          "Skepparvägen (Österåker kn)",
          "Svinninge Svinningeuddsvägen (Österåker kn)",
          "Nantes vägskäl (Österåker kn)",
          "Björnhammarsvägen (Österåker kn)",
          "Ardennervägen (Österåker kn)",
          "Svartgarnsvägen (Österåker kn)",
          "Karsvretavägen (Österåker kn)",
          "Hästängsuddsvägen (Österåker kn)",
          "Rydbo Saltsjöbad (Österåker kn)",
          "Prästtorpet (Österåker kn)",
          "Ubby gård (Österåker kn)",
          "Kulla vägskäl (Österåker kn)",
        ],
        "type": "JNY",
      },
      Object {
        "end": "2019-11-09T14:12:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Kulla vägskäl (Österåker kn)",
          "time": "13:56",
        },
        "stops": Array [
          "Kulla vägskäl (Österåker kn)",
          "Östra Ryds kyrka (Österåker kn)",
          "Kocktorp (Österåker kn)",
          "Roslags Näsby trafikplats E18 (Täby kn)",
          "Mörby centrum E18 (Danderyd kn)",
          "Danderyds sjukhus T-bana",
        ],
        "type": "JNY",
      },
      Object {
        "end": "2019-11-09T14:35:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Danderyds sjukhus T-bana",
          "time": "14:22",
        },
        "stops": Array [
          "Danderyds sjukhus T-bana",
          "Bergshamra T-bana (Solna kn)",
          "Universitetet T-bana (Stockholm kn)",
          "Tekniska Högskolan T-bana (Stockholm kn)",
          "Stadion T-bana (Stockholm kn)",
          "Östermalmstorg T-bana (Stockholm kn)",
          "T-Centralen T-bana (Stockholm kn)",
        ],
        "type": "JNY",
      },
      Object {
        "end": "2019-11-09T14:55:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Stockholm City station",
          "time": "14:46",
        },
        "stops": Array [
          "Stockholm City station",
          "Stockholm Södra station",
          "Årstaberg station (Stockholm kn)",
          "Älvsjö station (Stockholm kn)",
        ],
        "type": "JNY",
      },
      Object {
        "end": "2019-11-09T15:08:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Älvsjö station (Stockholm kn)",
          "time": "15:05",
        },
        "stops": Array [
          "Älvsjö station (Stockholm kn)",
          "Turingevägen (Stockholm kn)",
        ],
        "type": "JNY",
      },
    ],
    "start": Object {
      "date": "2019-11-09",
      "time": "13:34",
    },
  },
  Object {
    "duration": Object {
      "hours": 2,
      "minutes": 15,
    },
    "legs": Array [
      Object {
        "end": "2019-11-09T14:33:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Åkers Runö station (Österåker kn)",
          "time": "13:57",
        },
        "stops": Array [
          "Åkers Runö station (Österåker kn)",
          "Täljö station (Österåker kn)",
          "Rydbo station (Österåker kn)",
          "Hägernäs station (Täby kn)",
          "Viggbyholm station (Täby kn)",
          "Galoppfältet station (Täby kn)",
          "Täby centrum station",
          "Roslags Näsby station (Täby kn)",
          "Enebyberg station (Danderyd kn)",
          "Djursholms Ekeby station (Danderyd kn)",
          "Bråvallavägen station (Danderyd kn)",
          "Djursholms Ösby station (Danderyd kn)",
          "Mörby station (Danderyd kn)",
          "Universitetet station (Stockholm kn)",
          "Stockholm Östra station",
        ],
        "type": "JNY",
      },
      Object {
        "end": "2019-11-09T14:51:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Tekniska Högskolan T-bana (Stockholm kn)",
          "time": "14:45",
        },
        "stops": Array [
          "Tekniska Högskolan T-bana (Stockholm kn)",
          "Stadion T-bana (Stockholm kn)",
          "Östermalmstorg T-bana (Stockholm kn)",
          "T-Centralen T-bana (Stockholm kn)",
        ],
        "type": "JNY",
      },
      Object {
        "end": "2019-11-09T15:10:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Stockholm City station",
          "time": "15:01",
        },
        "stops": Array [
          "Stockholm City station",
          "Stockholm Södra station",
          "Årstaberg station (Stockholm kn)",
          "Älvsjö station (Stockholm kn)",
        ],
        "type": "JNY",
      },
      Object {
        "end": "2019-11-09T15:28:00",
        "start": Object {
          "date": "2019-11-09",
          "place": "Älvsjö station (Stockholm kn)",
          "time": "15:25",
        },
        "stops": Array [
          "Älvsjö station (Stockholm kn)",
          "Turingevägen (Stockholm kn)",
        ],
        "type": "JNY",
      },
    ],
    "start": Object {
      "date": "2019-11-09",
      "time": "13:57",
    },
  },
]
`;
